components/navbar/SideNav.js

```diff
@@ -62,8 +62,7 @@ export default function SideNav({ navigation }) {
                         {item.children.map((subItem) => (
                           <li key={subItem.name}>
                             {/* 44px */}
-                            <Disclosure.Button
-                              as="a"
+                            <Link
```

Comment:
I think it would be better to use `as` they have recommended in the docs 
https://headlessui.com/react/disclosure#rendering-a-different-element-for-a-component

Reply:
When using the 'as="a"' attribute, it results in a URL refresh. Thus far, I have not found a solution to this issue when using this approach. The common advice I've come across suggests using 'Link' instead of 'as="a"'. It seems that 'as="a"' does not work optimally in this specific scenario